app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -81,6 +102,28 @@ define(function (require, exports, module) {
       return ! account.has('profileImageUrl');
     },
 
+    _addLoadingSpinner: function ($wrapperClass) {
+      $wrapperClass.append('<span class="avatar-spinner"></span>');
+    },
+
+    // "Completes" the spinner, transitioning the semi-circle to a circle, and
+    // then removes the spinner element.
+    _completeLoadingSpinner: function ($wrapperClass) {
+      var deferred = p.defer();
+      var $spinner = $wrapperClass.find('.avatar-spinner');
```

Comment:
You are pointing out a deficiency in our conventions. We haven't been using `$` to indicate jQuery elements, we've (kind of sort of) been using, e.g., `spinnerEl`, which isn't very clear. I'm not adverse to the `$` prefix.

`$wrapperClass` is only used to find `$spinner`, might as well just pass in a reference to the spinner. If you don't, update the `$wrapperClass` name to something like `$spinnerWrapper`, because `Class` is an implementation detail of how the jQuery element is found, and doesn't describe the purpose of the element. I'd make the same name change in [_addLoadingSpinner](https://github.com/mozilla/fxa-content-server/pull/3324/files#diff-69c1b4c151f87a5c9ad048f78656cf9aR105).